src/modules/feature-modules/admin/pages/organisations/organisation-edit.component.ts

```diff
@@ -141,17 +136,16 @@ export class PageOrganisationEditComponent extends CoreComponent implements OnIn
   }
 
   errorResponse(error: { id: string }): void {
-    this.submitBtnClicked = false;
-
-    if (!this.securityConfirmation.id && error.id) {
-      this.securityConfirmation.id = error.id;
-      this.pageStep = 'CODE_REQUEST';
-
-    } else {
-
-      this.form.get('code')!.setErrors({ customError: true, message: 'The code is invalid. Please, verify if you are entering the code received on your email' });
-
+    switch (error.id) {
+      case 'O.0006':
+        this.alert = { type: 'ERROR', title: 'Organisation name or acronym already in use' };
+        break;
+      case 'O.0007':
+        this.alert = { type: 'ERROR', title: 'Organisation unit name or acronym already in use' };
+        break;
+      default:
+        this.alert = { type: 'ERROR', title: 'Error updating organisation' };
```

Comment:
It's better to use `OrganisationErrorsEnum` instead of the raw codes

Reply:
done